portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,11 +60,21 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         response.getWriter().println("Please enter a non-empty string");
         return;
       }
-      comments.addComment(comment);
+      sendToDatastore(comment);
       response.sendRedirect("/index.html");
   }
 
-  private String getComment(HttpServletRequest request) {
+  private void sendToDatastore(String comment){
+      DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+      long timestamp = System.currentTimeMillis();
```

Comment:
Nice job getting the timestamp! This is definitely a feature that goes beyond the MVP.